Makefile

```diff
@@ -162,14 +141,48 @@ full-reset: docker-compose-rm clean all
 
 .PHONY: docker
 docker:
-	docker build -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) .
+	docker build -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) --target=catalyst .
+
+.PHONY: docker
+docker-box: DOCKER_TAG=livepeer/in-a-box
+docker-box:
+	docker build -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) --target=livepeer-in-a-box .
 
 .PHONY: docker-local
-docker-local:
-	tar ch ./bin Dockerfile.local | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
+docker-box-local: DOCKER_TAG=livepeer/in-a-box-local
+docker-box-local: downloader livepeer-log scripts 
+	tar ch ./bin Dockerfile.local ./scripts ./config | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
 
+.PHONY: box-local
+box-local: DOCKER_TAG=livepeer/in-a-box scripts downloader livepeer-log
+box-local:
+	tar ch ./bin Dockerfile.local ./scripts ./config | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
+
+.PHONY: test
 test: docker
 	go test ./test/e2e/*.go -v --logtostderr
 
+.PHONY: test-local
 test-local: docker-local
 	go test ./test/e2e/*.go -v --logtostderr
+
+.PHONY: scripts
+scripts:
+	cp -Rv ./scripts/* ./bin
+
+.PHONY: box-dev
+box-dev: scripts
+	ulimit -c unlimited \
+	&& exec docker run \
+	-v $$HOME/code/livepeer-in-a-box-database-snapshots:/data \
+	-v $$(realpath bin):/usr/local/bin \
+	-v $$(realpath ../studio):/studio \
+	-v $$(realpath config):/config:ro \
+	-v $$(realpath ./coredumps):$$(realpath ./coredumps) \
+	-e CORE_DUMP_DIR=$$(realpath ./coredumps) \
+	--rm \
+	-it \
+	--name catalyst \
+	--shm-size=4gb \
+	--network=host \
```

Comment:
Works great for WebRTC on a Linux host but won't work on Docker for Mac. May need a port-range forward or may need to give up on WebRTC entirely. Will think on it.